### DISCUSSION TITLE
Intro to Github Flow task not working

### DISCUSSION THREAD

**Discussion by Rcorn36** [skills/.github#344] [Introduction to GitHub]
In the student training area the Intro to Github flow task does not work. I've tried every way from sunday. Once you create the new branch and make the file thats it. You cant go any further

**Comment by cmwilson21** [comment]
@Rcorn36 Was your branch titled `my-first-branch`? Some users have experienced problems when naming it something other than what's in the instructions. I've updated the [Skills course](https://github.com/skills/introduction-to-github) to reflect that stipulation. 

If you are following this course from Education, please open a discussion in the [Education Community](https://github.com/orgs/community/discussions/categories/github-education) ✨

**Comment by Rcorn36** [comment]
No I followed the directions to the t naming and everything. I tried it 500 different ways I swear. Anyways it's fine I'm going to skip the courses

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by cmwilson21** [reply]
If you run into any additional issues during an Skills course, please provide a link to your repo so the community can help debug 💖
